### ISSUE TITLE
React Version 

### ISSUE THREAD

**Issue by mrigaankzoro** [mrigaankzoro/ScholarStation#5]
Issue: Create Basic React Version of Scholar Station Website

Description: Develop a basic React version of the Scholar Station platform. Set up the initial React app with a simple structure, including basic components for the Home, Resources, and Contact pages. Ensure the app is functional, but keep the design minimal at this stage.

Ideal for beginners looking to contribute!

**Comment by AloneFFOfficial** [comment]
I will create a simple react prototype for this issue.

**Comment by mrigaankzoro** [comment]
Use vite bro

**Comment by bhatiatanish225** [comment]
As You said me to comment here , pls assign me this issue under hactoberfest-accepted tag i will create full responsive website for that in 2-3 days.
Thank You.

**Comment by AloneFFOfficial** [comment]
Ok I will be using react + vite + tailwind css combo for this.

**Comment by mrigaankzoro** [comment]
@bhatiatanish225 bro can u create the backend for it .I am assigning it to you too.

**Comment by me-safwan-07** [comment]
bro can we use the MERN Stack for this project so that i can maintain the backend !

**Comment by mrigaankzoro** [comment]
@me-safwan-07 yes we will use it @bhatiatanish225 @AloneFFOfficial  guys make a pr as soon as some work is completed like basic react templates is setup so make a pull request jisse aur bhi fr vhi Contribute kre pta chla sab alg bnane lg gye hai koi phele ek component bnake pr kr do fr koi navbar bna dega koi fotter ya page by page distribute krlo  

Isliye join the grp for discussion [https://chat.whatsapp.com/C9BzQpWPOqbJUj51N8cWr8](url) 

And provide your likendln in the gro too .

**Comment by AloneFFOfficial** [comment]
@mrigaankzoro Pull request sent with basic react setup

**Comment by mrigaankzoro** [comment]
Pull req mergeed 
Jo bhi kro changed ko dekhe krna

**Comment by bhatiatanish225** [comment]
@AloneFFOfficial Bro can u wait a little actually I have created a react app and did some code once I make a or then we will work on that

**Comment by bhatiatanish225** [comment]
I will make a pr by tonight

**Comment by bhatiatanish225** [comment]
@mrigaankzoro whatsapp link is not working,
Pls send the updated one.

**Comment by AloneFFOfficial** [comment]
@bhatiatanish225 ok bro will wait for ping me anytime when we have to resume the work.

**Comment by mrigaankzoro** [comment]
@bhatiatanish225  whatsapp link [][(url)](https://chat.whatsapp.com/C9BzQpWPOqbJUj51N8cWr8)

**Comment by mrigaankzoro** [comment]
@bhatiatanish225 ok bro we will wait 
Share your likendln will be posting the work till yet

**Comment by bhatiatanish225** [comment]
https://www.linkedin.com/in/tanish-bhatia-79a513249?utm_source=share&utm_campaign=share_via&utm_content=profile&utm_medium=android_app
